packages/core/src/browser/style/hover-service.css

```diff
@@ -17,7 +17,7 @@
 /* Adapted from https://github.com/microsoft/vscode/blob/7d9b1c37f8e5ae3772782ba3b09d827eb3fdd833/src/vs/workbench/services/hover/browser/hoverService.ts */
 
 :root {
-  --theia-hover-max-width: 500px;
+  --theia-hover-width: 300px;
```

Comment:
I believe this is another breaking change that adopters might depend on?
I don't think there is harm in keeping both variables (one for width and one for max-width).